tasks/flashfiles.mk

```diff
@@ -286,7 +303,7 @@ ISO_INSTALL_IMG = $(PRODUCT_OUT)/$(TARGET_PRODUCT)-sign-flashfile-$(FILE_NAME_TA
 else
 ISO_INSTALL_IMG = $(PRODUCT_OUT)/$(TARGET_PRODUCT)-flashfile-$(FILE_NAME_TAG).iso
 endif
-ISO_INSTALL_IMG_ZIP = $(ISO_INSTALL_IMG).zip
+ISO_INSTALL_IMG_ZIP = $(ISO_INSTALL_IMG).tar.gz
```

Comment:
better to rename ISO_INSTALL_IMG_ZIP to ISO_INSTALL_IMG_TAR_GZ

Reply:
I will make this change (y)